### ISSUE TITLE
[Epic]: DCR for Apps - Bridget video integration

### ISSUE THREAD

**Issue by jamesgorrie** [guardian/dotcom-rendering#8741]
We'll be using the current mechanism of creating a space in DCR, and then letting the native layer know where to put the video.

There is discussion as to whether we should be moving to using the native video vs the YouTube web APIs that we should probably catch up on before starting this work.

Considerations:
- Picture in picture
- Consent / ads

**Comment by JamieB-gu** [comment]
Closing in favour of #11942
